docs/usePermissions.md

```diff
@@ -61,6 +61,32 @@ const MyPage = () => {
 }
 ```
 
+## Refreshing permissions
+
+Permissions are loaded on page load and cached. If your application requires permissions to be refreshed, for example after a change modifying user permissions, you can use `refetch` function to trigger reload.
+
+```jsx
+const GrantAdminPermissionsButton = () => {
+    const record = useRecordContext();
+    const [ update ] = useUpdate();
+    const { refetch } = usePermissions();
+
+    const handleClick = () => {
+        update(
+            "users",
+            { id: record.id, data: { admin: true }, previousData: record },
+        );
+        refetch();
+    }
+
+    return (
+        <Button onClick={handleClick}>
+            Make user an admin
+        </Button>
+    )
+}
+```
```

Comment:
Also, this whole code block should be wrapped with `{% raw %}` `{% endraw %}` to avoid issues with jekyll. Can you wrap it?